test/AutoExpandbleMap/AutoExpandble_map_utest.cpp

```diff
@@ -0,0 +1,85 @@
+#include <gtest/gtest.h>
+
+#include "Map_test.h"
+
+TEST(AutoExpandbleMapBaseTests, Right) {
+  HeWi param = AutoExpandbleMap_Test({2, 0});
+  bool flag = (param.height == 1)&&(param.width == 4);
```

Comment:
the "flag" name is too generic, make it more specific. Apply this comment to the tests below.